test/unit/test_disk_tensor.py

```diff
@@ -128,32 +134,20 @@ def test_write_ones(self):
     out = reloaded.numpy()
     assert np.all(out == 1.)
 
-  def test_slice(self):
-    pathlib.Path(temp("dt3")).unlink(missing_ok=True)
-    Tensor.arange(10, device="CPU").to(f"disk:{temp('dt3')}").realize()
-
-    slice_me = Tensor.empty(10, device=f"disk:{temp('dt3')}")
-    print(slice_me)
-    is_3 = slice_me[3:4].cpu()
-    assert is_3.numpy()[0] == 3
-
-  def test_slice_2d(self):
-    pathlib.Path(temp("dt5")).unlink(missing_ok=True)
-    Tensor.arange(100, device="CPU").to(f"disk:{temp('dt5')}").realize()
-    slice_me = Tensor.empty(10, 10, device=f"disk:{temp('dt5')}")
-    tst = slice_me[1].numpy()
-    print(tst)
-    np.testing.assert_allclose(tst, np.arange(10, 20))
-
   def test_assign_slice(self):
-    pathlib.Path(temp("dt4")).unlink(missing_ok=True)
-    cc = Tensor.arange(10, device="CPU").to(f"disk:{temp('dt4')}").realize()
+    def assign_slice(x, slice, val):
+      x[slice] = val
+      return x[slice]
+    helper_test_disk_tensor("dt3", [0,1,2,3], lambda x: assign_slice(x, slice(0,2), [13, 12]), lambda x: x[0:2].assign([13, 12]))
```

Comment:
You shouldn't need a different numpy function here (or anywhere)

Reply:
yeah some like reshape dont need a new one - though I couldn't find an api like `.assign` or `.permute`